spec/models/inventory_type_spec.rb

```diff
@@ -1,5 +1,35 @@
 require 'rails_helper'
 
 RSpec.describe InventoryType, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it { 
+    is_expected.to have_many(:box_items)
+    .class_name('BoxItem'),
+    foreign_key: 'inventory_type_id',
+    inverse_of: :assembly_location
+  }
```

Comment:
Is this syntax correct? It doesn't seem to be.

Try changing the foreign key to another value and see if the spec will fail.